src/screens/HSwitchRemoteFilter.res

```diff
@@ -286,6 +286,7 @@ module RemoteTableFilters = {
         remoteOptions
         remoteFilters
         autoApply=false
+        submitInputOnEnter=true
```

Comment:
`submitInputOnEnter` this value need to be passed from `RemoteTableFilters` to enable customer ID filter enter press submit for orders. cureetly diresctly passed to FIlter compoment as true and RemoteTableFilters will enable it all all the places wherever its need used rather than only on orders page